Ext.data.JsonP.javax_security_auth_DestroyFailedException({"tagname":"class","name":"javax.security.auth.DestroyFailedException","autodetected":{},"files":[{"filename":"DestroyFailedException.js","href":"DestroyFailedException.html#javax-security-auth-DestroyFailedException"}],"extends":"java.lang.Exception","members":[],"alternateClassNames":[],"aliases":{},"id":"class-javax.security.auth.DestroyFailedException","short_doc":"Signals that a {@code destroy} operation failed. ...","classIcon":"icon-class","superclasses":["java.lang.Exception"],"subclasses":[],"mixedInto":[],"mixins":[],"parentMixins":[],"requires":[],"uses":[],"html":"<div><pre class=\"hierarchy\"><h4>Hierarchy</h4><div class='subclass first-child'>java.lang.Exception<div class='subclass '><strong>javax.security.auth.DestroyFailedException</strong></div></div><h4>Files</h4><div class='dependency'><a href='source/DestroyFailedException.html#javax-security-auth-DestroyFailedException' target='_blank'>DestroyFailedException.js</a></div></pre><div class='doc-contents'><p>Signals that a {@code destroy} operation failed.</p>\n\n<p> <p> This exception is thrown by credentials implementing\n the {@code Destroyable} interface when the {@code destroy}\n method fails.</p>\n</div><div class='members'></div></div>","meta":{}});